package.json

```diff
@@ -118,4 +122,4 @@
     "watch": "babel src --watch --out-dir lib --source-maps | node-sass -w src/styles/main.scss lib/react-chart-editor.css",
     "watch-test": "jest --watch"
   }
-}
+}
```

Comment:
please always put newlines at the ends of files, and don't remove them if they're already there :)